canonical-web-and-design/jaas-dashboard
# Store the remaining change deltas in the redux store.

## Done

Store remaining change deltas in the redux store.

## QA

Open up an existing model and inspect the redux store for juju/modelWatcherData and ensure that it has all of the top level entity keys... action, application, charm, machine, model, relation, unit.
